lms/templates/user_dropdown.html

```diff
@@ -14,22 +14,24 @@
 from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
 %>
 % if uses_bootstrap:
-    <li class="nav-item nav-item-hidden-collapsed">
-        <a href="${reverse('dashboard')}" class="menu-title">
-            <span class="sr-only">${_("Dashboard for:")}</span>
-            <% profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium'] %>
-            <img class="user-image-frame" src="${profile_image_url}" alt="">
-        </a>
-    </li>
-    <li class="nav-item dropdown mt-2 nav-item-hidden-collapsed">
-        <% username = self.real_user.username %>
-        <a class="nav-link dropdown-toggle" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">${username}</a>
-        <ul class="dropdown-menu dropdown-menu-right" id="${_("Usermenu")}" aria-labelledby="dropdownMenuLink" tabindex="-1">
-            <a class="dropdown-item" href="${reverse('dashboard')}">${_("Dashboard")}</a>
-            <a class="dropdown-item" href="${reverse('account_settings')}">${_("Account")}</a>
-            <a class="dropdown-item" href="${reverse('logout')}">${_("Sign Out")}</a>
-        </ul>
-    </li>
+    <div class="nav-item nav-item-hidden-collapsed container">
+        <div class="nav align-items-center">
+            <a class="menu-title mr-2" href="${reverse('dashboard')}">
+                <span class="sr-only">${_("Dashboard for:")}</span>
```

Comment:
I'm not sure why the username isn't added here. @cptvitamin shouldn't it be?

Reply:
It seems that it is because the username is shown as a link immediately following it.

Reply:
Again this isn't my change, so I'm not sure.

Reply:
Cool, I'm fine leaving as is.

Reply:
@andy-armstrong @AlasdairSwan The username needs to be included.  It used to be included using the alt attribute for the image that follows, but I see that image now has an empty alt attribute (no value).   It looks like this now:
```
<a href="/dashboard" class="user-link">
        <span class="sr">Dashboard for:</span>
        <img class="user-image-frame" src="..." alt="">
        <div class="label-username">username</div>
</a>
``` 
It has to be one or the other, either an alt attr value, or accessible text.